WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -125,23 +103,12 @@ public void onCreate(Bundle savedInstanceState) {
 
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.notifications_fragment_notes_list, container, false);
-
-        mRecyclerView = view.findViewById(R.id.recycler_view_notes);
+        View view = inflater.inflate(R.layout.notifications_list_fragment, container, false);
 
-        mActionableEmptyView = view.findViewById(R.id.actionable_empty_view);
         mConnectJetpackView = view.findViewById(R.id.connect_jetpack);
+        mToolbar = view.findViewById(R.id.toolbar_main);
+        mToolbar.setTitle(mToolbarTitle);
         mTabLayout = view.findViewById(R.id.tab_layout);
-        mTabLayout.addTab(mTabLayout.newTab().setText(getString(R.string.notifications_tab_title_all)),
-                TAB_POSITION_ALL);
-        mTabLayout.addTab(mTabLayout.newTab().setText(getString(R.string.notifications_tab_title_unread)),
-                TAB_POSITION_UNREAD);
-        mTabLayout.addTab(mTabLayout.newTab().setText(getString(R.string.notifications_tab_title_comments)),
-                TAB_POSITION_COMMENT);
-        mTabLayout.addTab(mTabLayout.newTab().setText(getString(R.string.notifications_tab_title_follows)),
-                TAB_POSITION_FOLLOW);
-        mTabLayout.addTab(mTabLayout.newTab().setText(getString(R.string.notifications_tab_title_likes)),
-                TAB_POSITION_LIKE);
         mTabLayout.addOnTabSelectedListener(new OnTabSelectedListener() {
```

Comment:
We might want to move the setting of `OnTabSelectedListener` after `setupWithViewPager` call, otherwise, the `TAB_POSITION_ALL` will be triggered every time configuration changes (eg. if you are on UNREAD tab, and rotate device the `TAB_POSITION_ALL` case will be called, and after that `TAB_POSITION_UNREAD`) .

Reply:
Nice catch!  That might be a better way to track usage of the tabs.  Interestingly, the original segmented control implementation tracks exactly the way you described.  We may want to keep the tracking as is in order to compare usage differences between the old segmented controls and new tabs layout directly.  What do you think?

Reply:
Yeah, If we want to be able to compare existing implementation with a new one we should probably keep the tracking as is. Maybe we can submit a fix in a month or so (however long it takes to get a reliable set of data for comparison).

Reply:
That sounds good to me.